ManikShrivastav.md

```diff
@@ -0,0 +1,45 @@
+---
+Name: “ManikShrivastav”
+Project: “Gesture Robot”
+---
+
+# Project Name
+Gesture Robot
+
+## Summary
+Gesture Robot is an electronic car controlled by the movement of circuit. It can be mostly used in real 
+life robot fights and car fights. It is a car that can be controlled according to the user’s movement of the 
+hands. The transmitter part will be attached to the hand of the user and its receiver will act as the 
+motherboard of the car. As the transmitter moves, it will send the signals to the receiver in the car and it 
+will turn and run the car according to the instructions.
+
+## Plan
+First of all, the chassis of car is going to be made. It will contain four motors and rubber wheels attached 
+to it. After that, the motors will be connected with motor shield driver. After that, the Arduino Nano is 
+connected to the RF module and it acts as a transmitter with the power supply of li-ion battery. The
+other Arduino Nano is connected with RF Module and ADLX335 Module and it acts as a receiver with 
+power supply from 9V Battery. The code would be uploaded to both the Arduinos to make them 
+transmit and receive signals and then the circuit works.
+
+## Budget
+
+| SN   | Product                               | Supplier/Link                         | Cost   |
+| ---- | ------------------------------------- |-------------------------------------- | ------ |
+|  1.  | Arduino Nano                          |  https://amzn.to/3GiAeTd              | 20.46  |
```

Comment:
```suggestion
|  1.  | 2x Arduino Nano                       |  https://amzn.to/3GiAeTd              | 20.46  |
```

It sounds like you'll be needing 2 of these?

Reply:
Yes, one for the robot and one for controlling the robot remotely.